tests/framework/matcher/phase_test.go

```diff
@@ -0,0 +1,35 @@
+package matcher
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("Matcher", func() {
```

Comment:
this is a unit test. right? The unit tests only execute if they are a part of this

```
bazel test \
    --config=${ARCHITECTURE} \
    --stamp \                                                                                                                  
    --test_output=errors -- //staging/src/kubevirt.io/client-go/... //pkg/... //cmd/...
```

this dir isn't listed. So does this unit test get executed as part of our functional tests?

Reply:
It is listed there: https://github.com/kubevirt/kubevirt/pull/4616/files#diff-c5d77fa510a26d6656083dd099dd8512f416243de60780c49c60d1c9ca2c3e7a

But it probably gets executed too on the functests. Need to fix that.

Reply:
No, all good there. They are executed in the unit test phase as expected, and not executed for e2e tests, since we don't compile them into the test binary.